core/openapi/schemas/folder.yaml

```diff
@@ -0,0 +1,61 @@
+---
+type: object
+required:
+  - id 
+  - type
+  - links
+properties:
+  id:
+    description:
+      An identifier of this folder.
+    type: string
+  type:
+    description:
+      Fixed value of "folder".
+    type:
+      string
+    enum:
+      - Folder
```

Comment:
If this would be "Catalog" the same entity could serve STAC and Records. With this proposal you need two entities side-by-side, which I think is not ideal. (Or would they never touch anyway?)

Reply:
No, the intention is that they would touch but the problem is that "catalogue" in OAPIR means a "collection or records" so we had a terminology problem and decided to use the term "folder" since it seemed like the function of a "STAC catalogue" was to act like a folder or directory for organizing records to be crawled (hopefully that interpretation is correct) .

A folder is a collection or references to any type of OAPIR resource including records, catalogues, other folders.  A collection (or record collection  or catalogue in OAPIR) strictly references records.

Folder has the following members:
- id, type(=Folder), title, description, keywords, created, updated, language, languages, extent, links.

Record collection had the following members:
- id, type(=Folder), title, description, keywords, created, updated, language, languages, extent, links.
- itemType, crs, conformsTo, themes, contacts, licence, right, properties.

Originally "folder" had 4 members: id, type(=Folder), title and links but then we got into a bit of a "it would nice if" game and started adding more stuff! LOL!

Reply:
Thanks, shifting the type discussion to https://github.com/opengeospatial/ogcapi-records/issues/264